chromium_src/chrome/browser/ui/webui/webui_util.cc

```diff
@@ -28,13 +28,17 @@ bool IsChromeUntrustedDataSource(content::WebUIDataSource* source) {
 }
 
 constexpr char kBraveCSP[] =
-    "script-src chrome://resources "
+    "script-src chrome://resources chrome://webui-test "
+#if BUILDFLAG(IS_CHROMEOS_ASH)
     "chrome://test "
+#endif
```

Comment:
Do we need this and the one below since it's specific to ChromeOS?

Reply:
I did that just to make it easier to scan quickly for upstream changes in the future. But yeah, it's not strictly necessary and I can remove if you think it's better that way.

Reply:
I usually skip things that are ChromeOS, but 🤷‍♂️

Reply:
Yeah, makes sense. Fixed in 2b04d4551b2d38de3cdb4a803072aead456955cd.